rpcs3/rpcs3qt/gui_application.cpp

```diff
@@ -354,6 +354,11 @@ void gui_application::InitializeCallbacks()
 		return m_gui_settings->GetBootConfirmation(m_main_window, gui::ib_confirm_fw);
 	};
 
+	callbacks.on_install_fw = [this]()
+	{
+		return m_main_window->InstallPup();
```

Comment:
this will crash in no-gui mode

Reply:
But callback is not called in no-gui mode.

Reply:
doesn't matter where it's called from.

Reply:
Fixed.